examples/TargetingConsoleApp/Program.cs

```diff
@@ -17,7 +17,7 @@
 services.AddSingleton(configuration)
         .AddFeatureManagement();
 
-IUserRepository userRepository = new InMemoryUserRepository();
+InMemoryUserRepository userRepository = new InMemoryUserRepository();
```

Comment:
```suggestion
var userRepository = new InMemoryUserRepository();
```

Reply:
Did we commit to this format with our linter? I believe others prefer this way, but personally, I don't like it. Haha.

Reply:
Both of these options are fine.

The only thing that's not fine is using var when the type isn't obvious in the assignment.

Reply:
For the sake of taking a direction and rolling with it, our code base heavily prefers the option of

`var userRepository = new InMemoryUserRepository();`

As the type repetition is unnecessary.

In the future I expect we'll shift to the latest language syntax for this scenario

`InMemoryUserRepository userRepository = new();`

Reply:
Sounds good, I'll do with `var` for now. I like the latest when we're able to make the switch.